lib_xua/api/xua_conf_default.h

```diff
@@ -1672,3 +1682,25 @@ enum USBEndpointNumber_Out
 #undef CHAN_BUFF_CTRL
 #endif
 
+/**
+ * @brief Enable Vendor specific control interface
+ *
+ * When enabled, device enumerates with an extra Vendor specific control interface with no associated endpoints
+ * Default: Disabled by default
+ */
+#ifndef USB_CONTROL_DESCS
+#define USB_CONTROL_DESCS    0
```

Comment:
Realise this wasn't your naming choice buy it ideally should be XUA_USB_CONTROL_DESCS

Reply:
XUA_USB_CONTROL_DESCS sounds more sensible. I'm wondering if this is a risky change in case there are apps using lib_xua that have defined USB_CONTROL_DESCS to 1. Updating to the new version would cause control etc. to mysteriously fail (unless ofcourse they read the changelog...)

Reply:
We should deffo add it to the changelog, also this is a major version bump